README.md

```diff
@@ -49,18 +49,30 @@ yet to be written.
   Layer), you need to understand HTML first.
   When learning HTML, you need to understand HTTP first.
 
+
 # Structure of Each Chapter / Section
 
 Each chapter or section (henceforth called generally *unit of writing*, abv.
 UOW) must have a certain structure.
 
-- chapters start with an abstract
-- UOW start with an estimated time-to-finish
-- some UOWs start with a motivation (answering the question *why learning
+- chapters start with an *`abstract`*
+- UOW start with an *`estimated time-to-finish`*
+- some UOWs start with a *`motivation`* (answering the question *why learning
   this is important?*)
-- some UOWs have a subdivision *Common Pitfalls* which contains exclusively
+- some UOWs have a subdivision `Common Pitfalls` which contains exclusively
   misunderstandings commonly observed among readers
-- some UOWs have a subdivision *Frequent Questions* which contains exclusively
+- some UOWs have a subdivision `Frequent Questions` which contains exclusively
   questions which have been asked more than twice by readers
-- exercises: ???
-- each chapter ends with a *Summary*
+- most UOWs should have a subvidision `TL;DR` which is placed near the
```

Comment:
`TL;DR`s would probably be overlapping a lot with `abstract`s, so I wouldn't recommend to add them.